### PR TITLE
Add command get-asset to get information on assets

### PR DESCRIPTION
**Pull request by mmitche**
Simple lookup for assets, useful for answering random questions

### CODE REVIEW

File: src/Microsoft.DotNet.Darc/src/DarcLib/MaestroApiBarClient.cs [hunk 1]
```diff
@@ -302,28 +302,9 @@ public async Task SetRepositoryMergePoliciesAsync(string repoUri, string branch,
...
-                        page: ++page);
-                    assets.AddRange(assetPage);
-                }
-                catch (RestApiException e) when (e.Response.StatusCode == HttpStatusCode.NotFound)
-                {
-                    break;
-                }
-            }
-            return assets;
+            PagedResponse<Asset> pagedResponse = await _barClient.Assets.ListAssetsAsync(name: name,
```
Comment (JohnTortugo) [inline src/Microsoft.DotNet.Darc/src/DarcLib/MaestroApiBarClient.cs]: NIT: not sure if these parameter names "name: name, version: version" are adding much value here :-)

File: src/Microsoft.DotNet.Darc/src/Darc/Options/GetAssetCommandLineOptions.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        public string Name { get; set; }
+
+        [Option("version", HelpText = "Look up specific version of an asset.")]
+        public string Version { get; set; }
+
+        [Option("channel", HelpText = "Look up the asset produced from builds applied to this channel.")]
+        public string Channel { get; set; }
+
+        [Option("max-lookup", Default = 10, HelpText = "Maximum number of builds to look up.")]
+        public int MaxBuildLookups { get; set; }
```
Comment (JohnTortugo) [inline src/Microsoft.DotNet.Darc/src/Darc/Options/GetAssetCommandLineOptions.cs]: Nit: wouldn't a `builds-after-date` parameter be more useful? :-)

Comment (mmitche) [reply src/Microsoft.DotNet.Darc/src/Darc/Options/GetAssetCommandLineOptions.cs]: Yeah, probably.  Or at least "days since". Will change.

**Comment by riarenas** [comment]
@mmitche I got a notification to review this again. Is there anything you want me to look at related to this?

**Comment by mmitche** [comment]
@riarenas Nope

**Pull request merged**